### PR TITLE
Updates per convo

### PR DESCRIPTION
**Pull request by shelleyolivia**
Noted clarifications around the public endpoint and the recommendation for private endpoints given throughput rates.

### CODE REVIEW

**Comment by shelleyolivia** [comment]
@Nicca42 @smartcontracts could I please get a review of this? I made updates based on the Alchemy convos, thanks so much!

File: src/docs/useful-tools/providers.md [hunk 1]
```diff
@@ -1,30 +1,40 @@
 ---
-title: Node Providers
+title: API Providers
```
Comment (Nicca42) [inline src/docs/useful-tools/providers.md:2]: Please change this back. Or compromise of:
Node & API Providers

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/useful-tools/networks.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Networks and Public RPC Endpoints
+title: Optimism APIs
```
[the remark this answers is not in the document]
Comment (shelleyolivia) [reply src/docs/useful-tools/networks.md:2]: Appreciate the context, sg we got feedback that devs look for APIs so I submitted the PR for "Networks, Public RPC Endpoints, & APIs". Thanks Vee!

File: src/docs/useful-tools/providers.md (hunk 1 above)
Comment (shelleyolivia) [reply src/docs/useful-tools/providers.md]: Changed to "Node & API Providers". Thanks Vee!

File: src/docs/useful-tools/networks.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 lang: en-US
 ---
 
-::: warning
-Some API calls, such as the those in the [personal namespace](https://geth.ethereum.org/docs/rpc/ns-personal) make no sense in a shared environment.
-Such RPCs are either totally unsupported, or will return nonsensical values.
+::: tip Developer Tip
```
[the remark this answers is not in the document]
Comment (shelleyolivia) [reply src/docs/useful-tools/networks.md:6]: Done, wasn't clear on that. Thanks Vee!

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**